public/app/percona/inventory/Tabs/Agents.tsx

```diff
@@ -51,8 +52,13 @@ export const Agents: FC<GrafanaRouteComponentProps<{ serviceId: string; nodeId:
   const { isLoading: nodesLoading, nodes } = useSelector(getNodes);
   const styles = useStyles2(getStyles);
 
+  const mappedNodes: Node[] = useMemo(
```

Comment:
No need to specify the result type here two times (or even once as it's inferred)